crates/bevy_input/src/mouse/unit.rs

```diff
@@ -0,0 +1,6 @@
+/// Unit of scroll
```

Comment:
I actually think this belongs in a `scrolling.rs` file instead, because then we can extend that file more readily.